desktop/main.js

```diff
@@ -80,6 +81,55 @@ const quitAndInstallWithUpdate = () => {
     autoUpdater.quitAndInstall();
 };
 
+/**
+ * Menu Item callback to triggers an update check
+ * @param {MenuItem} menuItem
+ * @param {BrowserWindow} browserWindow
+ */
+const manuallyCheckForUpdates = (menuItem, browserWindow) => {
+    // Disable item until the check (and download) is complete
+    // eslint: menu item flags like enabled or visible can be dynamically toggled by mutating the object
+    // eslint-disable-next-line no-param-reassign
+    menuItem.enabled = false;
+
+    autoUpdater.checkForUpdates()
+        .catch(error => ({error}))
+        .then((result) => {
+            const downloadPromise = result && result.downloadPromise;
+
+            if (downloadPromise) {
+                dialog.showMessageBox(browserWindow, {
+                    type: 'info',
+                    message: 'Update Available',
+                    detail: 'The new version will be available shortly. We’ll notify you when we’re ready to update.',
+                    buttons: ['Sounds good'],
+                });
+            } else if (result && result.error) {
+                dialog.showMessageBox(browserWindow, {
+                    type: 'error',
+                    message: 'Update Check Failed',
+                    detail: 'We couldn’t look for an update. Please check again in a bit!',
+                    buttons: ['Okay'],
+                });
+            } else {
+                dialog.showMessageBox(browserWindow, {
+                    type: 'info',
+                    message: 'Update Not Available',
+                    detail: 'There is no update available as of now! Check again at a later time.',
+                    buttons: ['Okay'],
+                    cancelId: 2,
+                });
+            }
```

Comment:
@kidroca 
Don't we need i18n here? 

cc: @pecanoro

Reply:
Right now it doesn't seem we have an established way to translate/localize texts used in `desktop/main` process 
You can see that other menus labels and texts used here are written in English as well 

Applying i18n to Desktop should be a separate task - I've mentioned this in the PR description:
> Note: As other content (menu items) in desktop/main.js the texts are not localized.
Dialogs, notifications, menus localization is out current scope